subprojects/docs/src/docs/userguide/migrating_from_ant.adoc

```diff
@@ -0,0 +1,328 @@
+
+ 1. _Configure_ — applies the configuration defined in the Ivy settings file
+ 2. _Resolve_ — locates the declared dependencies and downloads them to the cache if necessary
+ 3. _Retrieve_ — copies the cached dependencies to another directory
+
+Gradle's process is similar, but you don't have to explicitly invoke the first two steps. It performs them automatically. The third step doesn't happen at all — unless you create a task to do it — because Gradle typically uses the files in the dependency cache directly in classpaths and as the source for assembling application packages.
+
+Let's look at how Ivy's steps map to Gradle:
+
+Configuration::
+Most of Gradle's dependency-related configuration is baked into the build script, as you've seen with elements like the `dependencies {}` block. Another particularly important configuration element is link:{groovyDslPath}/org.gradle.api.artifacts.ResolutionStrategy.html[resolutionStrategy], which is accessible on dependency configurations. This provides many of the features you might get from Ivy's conflict managers and is a powerful way to control transitive dependencies and caching.
++
+Some Ivy configuration options have no equivalent in Gradle. For example, there are no lock strategies because Gradle ensures that its dependency cache is concurrency safe, period. Nor are there "latest strategies" because it's simpler to have a reliable, single strategy for conflict resolution. If the "wrong" version is picked, you can easily override it using forced versions or other resolution strategy options.
++
+See the chapters on <<managing_transitive_dependencies#managing_transitive_dependencies,Managing Transitive Dependencies>> and <<customizing_dependency_resolution_behavior#customizing_dependency_resolution_behavior,Customizing Dependency Resolution Behavior>> for more information on this aspect of Gradle.
+
+Resolution::
+Gradle will automatically resolve any dependencies you have declared and download them to its cache. It searches for dependencies in the repositories, with the search order defined by <<declaring_repositories#sec:declaring_multiple_repositories,the order in which the repositories declared>>.
++
+It's worth noting that Gradle supports the same dynamic version syntax as Ivy, so you can still use versions like `1.0.+`. You can also use the special `latest.integration` and `latest.release` labels if you wish. You can also configure the caching behavior for dynamic and changing versions via link:{groovyDslPath}/org.gradle.api.artifacts.ResolutionStrategy.html[resolutionStrategy].
+
+Retrieval::
+As mentioned, Gradle does not automatically copy files from the dependency cache. Its standard tasks typically use the files directly. If you want to copy the dependencies to a local directory, you can use something like this in your build script:
++
+[source,groovy]
+----
+task retrieveRuntimeDependencies(type: Copy) {
```

Comment:
Consider a real sample

Reply:
Real in what sense? If you mean I should convert it into a standard user manual sample that's tested and has Groovy & Kotlin DSL versions, then that's planned as explained in a comment above.

Reply:
Correct. I mean "real" just like the above comment suggesting a testable sample 👍